hypershift-operator/controllers/nodepool/kubevirt/kubevirt_test.go

```diff
@@ -238,12 +239,241 @@ func TestKubevirtMachineTemplate(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "Addtional networks are configured",
+			nodePool: &hyperv1.NodePool{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      poolName,
+					Namespace: namespace,
+				},
+				Spec: hyperv1.NodePoolSpec{
+					ClusterName: clusterName,
+					Replicas:    nil,
+					Config:      nil,
+					Management:  hyperv1.NodePoolManagement{},
+					AutoScaling: nil,
+					Platform: hyperv1.NodePoolPlatform{
+						Type: hyperv1.KubevirtPlatform,
+						Kubevirt: generateKubevirtPlatform(
+							memoryNPOption("5Gi"),
+							coresNPOption(4),
+							imageNPOption("testimage"),
+							volumeNPOption("32Gi"),
+							additionalNetworksNPOption([]hyperv1.KubevirtNetwork{
+								{
+									Name: "ns1/nad1",
+								},
+								{
+									Name: "ns2/nad2",
+								},
+							}),
```

Comment:
do we really need all these values ? 

Wouldn't it be simpler to just put the relevant kubevirt platform configuration ? (the `additionalNetworks`)

Reply:
ditto